### ISSUE TITLE
go std::bind()调度方式存在内存泄漏？

### ISSUE THREAD

**Issue by hardeagle** [yyzybb537/libgo#262]
同样的逻辑代码,使用不同方式（协程调度，线程调度）
go std::bind();
std::thread(std::bind()).detach()
创建5k的并发调度，结束调度函数时，go调度占用内存不变，而thread调度占用的内存规范
是使用go调度方法不对，还是此协程库中有内存泄漏？
